newrelic-php-agent/Dockerfile

```diff
@@ -1,11 +1,11 @@
 FROM alpine:3.6
```

Comment:
Why do you use 3.6 ? Latest image tag is 3.10.

https://hub.docker.com/layers/alpine/library/alpine/3.10/images/sha256-6dff84dbd39db7cb0fc928291e220b3cff846e59334fd66f27ace0bcfd471b75

Reply:
@cw-sakamoto Change to that tag. Thank you!!